models/engine/file_storage.py

```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """Deserialize JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Returns obj based on cls and id else None"""
+        return self.__objects.get(cls + '.' + id) \
```

Comment:
If this class and id combo don't exist this will generate a key error, unless the default return value of None is specified